keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -173,6 +179,19 @@ Windows nodes will adhere to Kubernetes version-skew policy (node to control pla
 - [cncf-k8s-conformance thread](https://lists.cncf.io/g/cncf-k8s-conformance/topic/windows_conformance_tests/27913232)
 - [kubernetes/enhancements proposal](https://github.com/kubernetes/features/issues/116)
 
+### API Changes Proposed
+
+#### Add V1.SecurityContext.WindowsOptions.runAsUsername
```

Comment:
There are pod-level and container-level securityContext stanzas. Both allow setting runAsUser for linux today (container-specific security context overrides pod-level). The GMSA design also described being able to set the credentialSpec pod-wide and per-container. Would `windowsOptions.runAsUsername` be added to both the pod and container-level `securityContext` structs?